bin/build_daq

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+
+tag=
+
+if [ "$1" == "dts" ]; then
+  tag="dts"
+fi
+
+if [ -z "$tag" ]; then
```

Comment:
this is not really an image tag; it's part of the image name.

Reply:
Changed to image_name